javascript/sdk/lib/controllers/detect.ts

```diff
@@ -59,8 +60,12 @@ export default class DetectController {
   }
 
   /** Get report for an Account */
-  async describeActivity(days: number = 7, options: RequestOptions = {}) {
-    const searchParams = new URLSearchParams({ days: days.toString() });
+  async describeActivity(
+    start: string,
```

Comment:
both are optional, just so you know. defaults to start=30 days ago, finish = right now.